client/post-editor/post-editor.jsx

```diff
@@ -528,7 +532,9 @@ const PostEditor = React.createClass( {
 				if ( res.status === 200 ) {
 					html = JSON.parse( res.text ).html;
 				}
-				callback( html );
+				callback( html.replace( /((?:\\.|[^"\\])*).png/g, function( url ) {
```

Comment:
I have this regex to replace relative URLs for images to where `mozmaker-templates` is hosting the file until @gvn have the fix for that.